Dockerfile.foreman

```diff
@@ -0,0 +1,18 @@
+FROM python:3.5-slim
+
+RUN groupadd user && useradd --create-home --home-dir /home/user -g user user
+WORKDIR /home/user
+
+ENV CELERY_VERSION 4.0.2
+
+COPY workers/requirements.in .
```

Comment:
The `Dockerfile.foreman` name here is a bit confusing because it's not actually doing any `foreman` stuff yet, it's just sending a celery message from a container. I could probably rename this `Dockerfile.tester` to clear up confusion.